dom-weather-app/public/app.js

```diff
@@ -0,0 +1,91 @@
+const cityInput = document.querySelector('#city-input');
+const temperature = document.querySelector('.temperature');
+const dayOrNight = document.querySelector('.time');
+const conditions = document.querySelector('.conditions');
+const icon = document.querySelector('.weather-icon');
+const errorMessage = document.querySelector('#error');
+
+const apiKey = 'qGLMGbgG2tHUnaRGFoJIvHMul7t55Esr';
```

Comment:
normalnie taki sercret by był na backendzie i to np backend może robić 2 requesty, a FE tylko 1 do BE z nazwą miejscowości. Wtedy BE np mógłby cachować odpowiedzi jak jest duzo ruchu

Reply:
Spróbuje doczytać jak takie coś się robi - na razie poprawiłem tak, żeby na samym FE było ładnie